src/widgettoolbarrepository.js

```diff
@@ -243,7 +243,7 @@ function getBalloonPositionData( editor, relatedElement ) {
 	const defaultPositions = BalloonPanelView.defaultPositions;
 
 	return {
-		target: editingView.domConverter.viewToDom( relatedElement ),
+		target: editingView.domConverter.mapViewToDom( relatedElement ),
```

Comment:
We need a test for this change.

Reply:
I'm trying to write a test for this. And the only case when this fails if the `getRelatedElement()` will return an object with API similar to view element and the `viewToDom` will try to create a DOM node and somehow the domDocument is null. I'm lost how to write a test for this.

Also, this issue is really about the wrong method use with wrong parameters. Also changing `viewToDom` to proper `mapViewToDom` does not fix the issue with wrong API use.